.ci/verify

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd "$(dirname $0)/.."
+
+git config --global user.email "gardener@sap.com"
+git config --global user.name "Gardener CI/CD"
+
+# Required due to https://github.com/kubernetes/kubernetes/issues/86753 - can be removed once the issue is fixed.
+mkdir -p /go/src/github.com/gardener/gardener-resource-manager
+cp -r . /go/src/github.com/gardener/gardener-resource-manager
+cd /go/src/github.com/gardener/gardener-resource-manager
```

Comment:
Rather use $GOPATH to be sure?

Reply:
We do it like this in all the extensions, e.g. https://github.com/gardener/gardener-extension-provider-aws/blob/master/.ci/verify#L13-L15, I would like to keep it like this for now to make the pipeline happy.